lib/util/git.js

```diff
@@ -19,9 +19,9 @@ const tryGit = async (path, ...args) => {
 
 // parse a repo from a git origin into a format
 // for a package.json#repository object
-const getUrl = async (path) => {
+const getRemoteUrl = async (path, remote) => {
   try {
-    const urlStr = await tryGit(path, 'remote', 'get-url', 'origin')
+    const urlStr = await tryGit(path, 'remote', 'get-url', remote)
```

Comment:
```sh
~/D/n/c/b/main (latest|✔) $ git remote get-url upstream
error: No such remote 'upstream'
~/D/n/c/b/main (latest|✔) $ echo $status
2
```
does this gracefully handle if it's missing?

Reply:
yeah, it will return undefined if its missing and then move to the next remote